repository.datadog.yml

```diff
@@ -14,4 +14,15 @@ checks_to_ignore:
   - "Send PR Approval Status"
   - "Preview Synthetics"
   - "Documentation Team Approval Check / check-documentation-team-approval (pull_request)"
-  - "Documentation Team Approval Check / check-documentation-team-approval (pull_request_review)"
\ No newline at end of file
+  - "Documentation Team Approval Check / check-documentation-team-approval (pull_request_review)"
+---
+schema-version: v1
+kind: mergegate
+rules:
+  - require: commit-signatures
+    enforcement: warn
+    excluded_emails:
+      - 'robot-guacbot-guacamole@datadoghq.com'
+      - 'guacbot@users.noreply.github.com'
+      - '41898282+github-actions[bot]@users.noreply.github.com'
+    allow_unsigned_external: true
```

Comment:
This line allows unsigned commits from external contributors (ie, if that's the only source of an unsigned commit in a PR a warning will not be generated)